salt/iiif/config/lib-systemd-system-iiif-service.service

```diff
@@ -6,10 +6,12 @@ Before=nginx.service
  
 [Service]
 TimeoutStartSec=0
-Restart=always
-
 WorkingDirectory=/opt/loris
-ExecStart=/usr/local/bin/docker-compose up
+ExecStart=/usr/local/bin/docker-compose up --detach
+ExecStop=/usr/local/bin/docker-compose down
+# https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=
+Type=oneshot
```

Comment:
this is a bit mysterious to me but it works like a charm and is being recommended as a way for systemd and docker-compose to inter-operate.